Makefile

```diff
@@ -101,6 +101,15 @@ test_e2e:
 	hack/e2e-test.sh
 .PHONY: test_e2e
 
+# Build and run node end-to-end tests.
+#
+# Example:
+#   make test_e2e_node
+test_e2e_node:
+	sudo echo "" && hack/e2e-node-test.sh
```

Comment:
Since this just breaks the make target and the tests can still be run directly, I'll leave sudo -v.